core/src/main/java/com/sequenceiq/cloudbreak/controller/mapper/ConstraintViolationExceptionMapper.java

```diff
@@ -1,14 +1,16 @@
 package com.sequenceiq.cloudbreak.controller.mapper;
 
-import java.util.ArrayList;
-import java.util.List;
-
 import javax.validation.ConstraintViolation;
 import javax.validation.ConstraintViolationException;
 import javax.ws.rs.core.Response.Status;
 
 import org.springframework.stereotype.Component;
 
+import com.sequenceiq.cloudbreak.json.ValidationResult;
+
+/**
```

Comment:
This comment is unnecessary, because class declaration tells the same :)

Reply:
I'll add some more information to make it useful.

Reply:
We don't prefer comments at all. Except a few unique situations. Comments are good if they are
- explaining for example why we need a workaround or bad code
- describing a public API interface

Otherwise they are just maintainable part of the code which loose consistency with code easily. Code is the documentation :).

Reply:
I've removed the header comment ... for now. ;)